mouse/record.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+
+
+import argparse
+import io
+import time
+import picamera
+import datetime
+import os
+
+
+
+def main(*argv):
+   """
+       Defines arguments: time before and after, and mouse/trial to be tested
+
+   """
+   
+   
+   parser = argparse.ArgumentParser(description='seconds before and after lever is pressed.')
+   parser.add_argument('before', type=int, help='seconds to record before.')
+   parser.add_argument('after', type=int, help='seconds to record after.')
+   parser.add_argument('folder', type=str, default=".", help = 'mouse to be tested')
```

Comment:
Quotes are inconsistent. Use double quotes for multiple characters. Use single quotes for a single character.